prisma/migrations/20240513040814_add_settings_table/migration.sql

```diff
@@ -0,0 +1,24 @@
+-- CreateEnum
+CREATE TYPE "ChannelSyncOptions" AS ENUM ('clientAndCompany');
+
+-- CreateTable
+CREATE TABLE "Settings" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "workspaceId" VARCHAR(32) NOT NULL,
+    "internalUserId" UUID NOT NULL,
+    "bidirectionalSlackSync" BOOLEAN NOT NULL DEFAULT false,
+    "channelsToSync" "ChannelSyncOptions" NOT NULL DEFAULT 'clientAndCompany',
+    "fallbackMessageSenderId" UUID NOT NULL,
+    "slackChannelPrefix" VARCHAR(255) NOT NULL,
+    "isSyncing" BOOLEAN NOT NULL DEFAULT false,
```

Comment:
can we accomodate this column with the `syncStatus` column?
```
enum SyncStatus {
 PENDING,
 SUCCESS,
 FAILED
}
```

Reply:
in case of failed, maybe also added errors column

Reply:
n/a